src/test/java/com/puppycrawl/tools/checkstyle/BaseCheckTestSupport.java

```diff
@@ -19,449 +19,11 @@
 
 package com.puppycrawl.tools.checkstyle;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
+/**
+ * @deprecated Checkstyle will not support BaseCheckTestSupport anymore. Use
+ *             {@link BaseTestSupport} instead.
+ */
+@Deprecated
+public class BaseCheckTestSupport extends BaseTestSupport {
```

Comment:
But then I am getting a checkstyle error -> `checkstyle/BaseCheckTestSupport.java:26:1: Name 'BaseCheckTestSupport' must match pattern '^Abstract.+$'. [AbstractClassName]`. So this means that I would have to name it `AbstractBaseCheckTestSupport` and that would defeat the purpose to make wercker build pass.